llm/requirements.txt

```diff
@@ -4,6 +4,8 @@ torch==1.12.1
 mosaicml==0.11.1
 mosaicml-streaming==0.1.2
 flash-attn==0.2.2
+# need a newer version of triton
+triton==2.0.0.dev20221103
```

Comment:
any particular reason for this version vs. `2.0.0.dev20221202`

Reply:
copy pasta [req from bert](https://github.com/mosaicml/examples/blob/main/bert/requirements.txt#L9)